Extensions/DraggableBehavior/draggableruntimebehavior.ts

```diff
@@ -193,7 +193,10 @@ namespace gdjs {
       }
       const position = this.getPosition(runtimeScene, draggableRuntimeBehavior);
       if (
-        !draggableRuntimeBehavior.owner.insideObject(position[0], position[1])
```

Comment:
Nice! Would be good to change the JS documentation of `insideObject` to tell that "`isCollidingWithPoint` is available if you want to check if a point is inside the object collision mask."

Reply:
Actually, I think it may be worth to do first a AABB check (`insideObject`) and then a precise check (`isCollidingWithPoint`).

Otherwise, we risk to run complex code (`isCollidingWithPoint`) for potential a lot of objects (if noted as `N`, and complexity of the point-polygon collision check as `C`, it results in `N*C`).
With `insideObject`, the checks should be simpler, so the cost is "only" `N*c` (`c` being smaller than `C`).

A really good solution would be to have a way to check objects that are in a certain area (which is possible if storing objects in a spatial datastructure like a R-Tree, giving a search in `O(log N)`)... but for now let's not implement that because it would need to be offered by the game engine itself I think.

Reply:
Alright !